<launch>
  <!-- Debug Info -->
  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value="" />
  <arg     if="$(arg debug)" name="launch_prefix" value="gdb --ex run --args" />

  <include file="$(find restorebot_moveit_desc)/launch/planning_context.launch">
    <arg name="load_robot_description" value="true"/>
  </include>

  <node pkg="tf" type="static_transform_publisher" name="virtual_joint_broadcaster_0" args="0 0 0 0 0 0 world base_footprint 100" />

  <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher">
    <param name="/use_gui" value="true"/>
  </node>

  <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" respawn="true" output="screen" />

  <include file="$(find restorebot_moveit_desc)/launch/moveit_rviz.launch">
  </include>

<node name="fmt_shortcut_restorebot" pkg="restorebot_moveit_desc" type="fmt_shortcut_restorebot" respawn="false" launch-prefix="" output="screen">
    <param name="allowed_planning_time" value="7" />
    <param name="max_Iter" value = "100" />
    <param name="max_EdgeLength_Discretization" value="0.1" />
    <param name="max_EdgeLength_Waypoint_Injection" value="0.2" />
    <param name="sbp_plugin" value="ompl_interface/OMPLPlanner"/>
    <rosparam command="load" file="$(find restorebot_moveit_desc)/config/kinematics.yaml"/>
    <rosparam command="load" file="$(find restorebot_moveit_desc)/config/ompl_planning.yaml"/>
  </node>

</launch>

